doc/visual-programming/source/widgets/evaluate/liftcurve.md

```diff
@@ -7,31 +7,28 @@ Measures the performance of a chosen classifier against a random classifier.
 
 - Evaluation Results: results of testing classification algorithms
 
-The **Lift curve** shows the relation between the number of instances which were predicted positive and those that are indeed positive and thus measures the performance of a chosen classifier against a random classifier. The graph is constructed with the cumulative number of cases (in descending order of probability) on the x-axis and the cumulative number of true positives on the y-axis. Lift curve is often used in segmenting the population, e.g., plotting the number of responding customers against the number of all customers contacted. You can also determine the optimal classifier and its threshold from the graph.
+The **Lift curve** shows to curves for analysing the proportion of true positive data instances in relation to the classifier's threshold or the number of instances that we classify as positive.
+
+Cummulative gains chart shows the proportion of true positive instances (for example, the number of clients who accept the offer) as a function of the number of positive instances (the number of clients contacted), assuming the the instances are ordered according to the models probability of being positive (e.g. ranking of clients).
```

Comment:
according to the models --> according to the model's

Reply:
Also, cumulative is written with a single m.

Reply:
... which I occasionally, but seldom do.